ChildProject/pipelines/eafbuilder.py

```diff
@@ -51,6 +53,37 @@ def create_eaf(
         )
         eaf.add_annotation("context_" + eaf_type, context_onset, context_offset)
 
+    if speech_segments is not None:
+
+        from ChildProject.converters import AnnotationConverter
+
+        type_to_id = {
+            val: key
+            for key, val in reversed(
+                list(AnnotationConverter.SPEAKER_ID_TO_TYPE.items())
+            )
+        }
+
+        for segment in speech_segments.to_dict(orient="records"):
+            speaker_id = None
+
+            if "speaker_id" in segment:
```

Comment:
This won't work for the LENA, because LENA defines speaker_id, but the values are not commensurable to that of our eaf templates.

Reply:
This has been changed, it directly takes the LENA label as the tier name